mantidproject/mantid
# Normalization by bin width in muon analysis

The asymmetry in muon analysis was being normalized to bin width for transverse field data (MUSR), however this should not occur. This fix forces the normalization to be off if the data is being presented as an asymmetry. 

**To test:**


With a previous version of mantid open muon analysis and load some MUSR data. Make sure that the plot type is asymmetry. Then in the settings tab change the "Rebin data" to different fixed values, you should see the amplitude of the asymmetry change (such that changing Rebin data from 1 to 10 will cause a factor 10 reduction in the asymmetry).  The other plot type should not show this behavior. 

In this version of Mantid load the same data set as above and repeat the tests. The asymmetry should not change amplitude, unless the data is rebinned too coarsely.  
Fixes #19294 .



---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
